src/coreclr/tools/aot/ILCompiler.Reflection.ReadyToRun/GCInfoTypes.cs

```diff
@@ -154,6 +154,11 @@ internal GcInfoTypes(Machine machine)
                     STACK_BASE_REGISTER_ENCBASE = 2;
                     NUM_REGISTERS_ENCBASE = 3;
                     break;
+                case (Machine)0x5064: /* TODO: update with RiscV64 */
```

Comment:
Same question as before. Do we have a tracking work item to change these hardcoded values to the respective enum values?